homeassistant/components/aurora_abb_powerone/aurora_device.py

```diff
@@ -25,12 +23,9 @@
 class AuroraEntity(Entity):
```

Comment:
You can use `CoordinatorEntity` as a base class which means you don't have to implement `available`

Reply:
Yes, that was already in use at the next level down within `sensor.py::AuroraSensor`.

But now that I look at it, this whole class is unnecessary, so I've removed it.